src/Storages/NATS/NATSHandler.cpp

```diff
@@ -43,7 +43,7 @@ void NATSHandler::startLoop()
 
     while (loop_state.load() == Loop::RUN && duration.count() < MAX_THREAD_WORK_DURATION_MS)
     {
-        uv_run(loop, UV_RUN_NOWAIT);
+        uv_run(loop, UV_RUN_DEFAULT);
```

Comment:
Because of this line, to be precise

Reply:
Sorry. It's sad, any ideas on how to fix this?